src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionVerticle.java

```diff
@@ -399,6 +404,58 @@ private void getImageData(Message<String> message) {
         }
     }
 
+    private void getHistogramJson(Message<String> message) {
+        ObjectMapper mapper = new ObjectMapper();
+        HistogramCtx histogramCtx;
+        try {
+            histogramCtx = mapper.readValue(message.body(), HistogramCtx.class);
+        } catch (Exception e) {
+            String v = "Illegal image region context";
+            log.error(v + ": {}", message.body(), e);
+            message.fail(400, v);
+            return;
+        }
+        ScopedSpan span = Tracing.currentTracer().startScopedSpanWithParent(
+                "get_histogram",
+                extractor().extract(histogramCtx.traceContext).context());
+        String omeroSessionKey = histogramCtx.omeroSessionKey;
+        log.debug("Get image data request: {}", histogramCtx.toString());
+        try (OmeroRequest request = new OmeroRequest(
+                host, port, histogramCtx.omeroSessionKey))
+       {
+           if (families == null) {
+               request.execute(this::updateFamilies);
```

Comment:
I think they aren't necessary anymore. I had originally thought we were going to provide a histogram of the rendered pixel values, but since we're only returning raw pixel values these probably aren't necessary.